encode.go

```diff
@@ -14,6 +14,27 @@ func (p sortValues) Len() int           { return len(p) }
 func (p sortValues) Less(i, j int) bool { return p[i].String() < p[j].String() }
 func (p sortValues) Swap(i, j int)      { p[i], p[j] = p[j], p[i] }
 
+type sortFields []reflect.StructField
```

Comment:
Did you reintroduce `sortFields` by mistake?

Reply:
Oh, you're right. That's indeed a mistake. I'll remove it once again!